packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -352,14 +358,39 @@ skipInBrowser(describe)('plugin-meetings', () => {
         });
       });
       describe('#createMeeting', () => {
+        let destination;
+
         beforeEach(() => {
+          destination = 'Y2lzY29zcGFyazovL3VzL1BFT1BMRS8wMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDA';
           MediaUtil.createPeerConnection = sinon.stub().returns(true);
           webex.internal.device.userId = uuid1;
           webex.internal.device.url = url1;
           MeetingCollection.set = sinon.stub().returns(true);
           MeetingsUtil.getMeetingAddedType = sinon.stub().returns('test');
           MeetingsUtil.extractDestination = sinon.stub().returns('test');
         });
+
+        it('should accept hydra personId values', async () => {
+          const personUUID = '00000000-0000-0000-0000-000000000000';
+          const meeting = await webex.meetings.createMeeting(destination);
+
+          assert.equal(meeting.destination, personUUID);
+        });
+
+        it('should call MeetingUtil#determineIfDestinationIsPerson()', async () => {
+          // Store the existing function property to be reassigned at end of test.
+          const {determineIfDestinationIsPerson} = MeetingsUtil;
+
+          MeetingsUtil.determineIfDestinationIsPerson = sinon.stub().returns(false);
+
+          await webex.meetings.createMeeting(destination);
+
+          assert.calledOnce(MeetingsUtil.determineIfDestinationIsPerson);
+
+          // Reassigned stored function property.
+          MeetingsUtil.determineIfDestinationIsPerson = determineIfDestinationIsPerson;
```

Comment:
Using sinon's `restore` functionality should work:
```suggestion
          MeetingsUtil.determineIfDestinationIsPerson.restore();
```

Reply:
implemented.